dockers/docker-fpm-frr/frr/supervisord/supervisord.conf.j2

```diff
@@ -77,7 +77,7 @@ dependent_startup_wait_for=zebra:running
 {% endif %}
 
 [program:bgpd]
-command=/usr/lib/frr/bgpd -A 127.0.0.1 -M snmp
+command=/usr/lib/frr/bgpd -A 127.0.0.1 -M snmp -M bmp
```

Comment:
>-M bmp [](http://example.com/codeflow?start=47&length=6)

If the feature is disabled, this change still has performance impact or maybe regression to FRR. Please consider FEATURE table for j2 rendering.